### PR TITLE
build: Update to Go 1.20, Alpine 3.17 and linter v1.51.2

### PR DESCRIPTION
**Pull request by lenny-intel**
**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A**
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A**
  <link to docs PR>

## Testing Instructions
run non-secure edgex stack  
build and run service from this branch 
verify service bootstraps w/o error

### CODE REVIEW

File: snap/local/helper-go/go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module github.com/edgexfoundry/app-rfid-llrp-inventory/local/helper-go
 
-go 1.18
+go 1.208
```
Comment (marcpfuller) [inline snap/local/helper-go/go.mod:3]: I think this was a typo
```suggestion
go 1.20
```

**Review by jim-wang-intel — APPROVED** [review]
other than marc's comment, LGTM

File: snap/local/helper-go/go.mod (hunk 1 above)
Comment (lenny-intel) [reply snap/local/helper-go/go.mod:3]: Goo catch!

Comment (lenny-intel) [reply snap/local/helper-go/go.mod]: fix pushed

**Review by marcpfuller — APPROVED** [review]

**Pull request merged**